bin/cli.js

```diff
@@ -75,7 +76,8 @@ const spinner = ora({
 	text: 'Finding files',
 	color: 'white',
 	interval: 100000,
-	isEnabled: program.spinner
+	isEnabled: !ci.isCI && program.spinner,
+	isSilent: ci.isCI
```

Comment:
I misread the ci-info package by simply suggesting it, but after checking [ora's description of the isEnabled option](https://github.com/sindresorhus/ora#isenabled), it seems that there are considerations in the package. 

> Force enable/disable the spinner. If not specified, the spinner will be enabled if the stream is being run inside a TTY context (not spawned or piped) and/or not in a CI environment.


On the source code, it is described as [this](https://github.com/sindresorhus/ora/blob/476935f318868265303d148992fc268639a0d573/index.js#L123) and It seems to be designed to avoid undefined, null, etc. when they are passed.

This problem may be fixed by passing null to isEnabled except when program.spinner is true, as shown below.

May I ask you to verify if the following code will work?

```suggestion
	isEnabled: program.spinner === false ? false : null,
```

Reply:
It did work but I had to make one other change as well. Here we were checking against the `program.spinner` if we should render a spinner and firing it manually (via `render()`).

```js
if (program.spinner) {
    spinner.text = chalk.grey(dir) + chalk.cyan(file);
    spinner.render();
}
```

I changed that to:
```js
spinner.text = chalk.grey(dir) + chalk.cyan(file);
```
and left the spinner to figure out if we should do it itself.

Let me know what do you think about that.

P.S. I was testing this with `npm run generate &> file.txt` since `ora` is using another [package](https://github.com/sindresorhus/is-interactive/blob/main/index.js) to determine if the environment is interactive.

Reply:
Hey, @kamiazya. Sorry to bother you. Have you had the chance to look over this? :)

Reply:
Sorry, I have been busy with work and have not been able to check the operation.
I am aware of this, so please wait a moment :pray:

Reply:
Just anxious to make my first OSS contribution. Sorry for bothering you 🙏